### PR TITLE
Remove Flickering Dots

### PR DESCRIPTION
**Pull request by iqrashahzad14**
New Dots start at timepoint 0

### CODE REVIEW

File: src/dot/reseedDots.m [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
         dots.positions(N, :) = positions;
         dots.speeds(N, :) = speeds;
-        dots.time(N, 1) = time;
```
Comment (Remi-Gau) [inline src/dot/reseedDots.m:38]: Makes sense to me to do this indeed.

But now that we don't use the `time` variable, you can remove it from line 34 and only keep:

```
[positions, speeds] = seedDots(dots, cfg, isSignal);
```

**Review by Remi-Gau — CHANGES_REQUESTED** [review]
Looks good to me. Just one tiny thing to change to keep things clean.

Also for next time, create a new branch starting from `dev` and make your changes on this branch.

Otherwise your pull request shows a lot more changes than what you actually did and it is confusing to review.

If that's unclear I can explain it next time I see you.

**Comment by marcobarilari** [comment]
@iqrashahzad14 thank you very much to catch this one, it is very useful (sorry to take over, we might use this patch tomorrow ;) )

File: src/dot/reseedDots.m (hunk 1 above)
Comment (marcobarilari) [reply src/dot/reseedDots.m:38]: So, now line 34 is clean. Though I am not confident to delete the `time` calculation in `seedDots`. I am afraid to break things if this is used in another part. (We do not have a dependency visualizer tool yet, do we?)

https://github.com/cpp-lln-lab/CPP_PTB/blob/192b180ad6483adb9f070225dcee9693de342a98/src/dot/seedDots.m#L25-L28

**Review by marcobarilari — APPROVED** [review]

**Comment by marcobarilari** [comment]
tested and working, merging

**Pull request merged**